### ISSUE TITLE
I have a question about your code!

### ISSUE THREAD

**Issue by Gepeng18** [yh1226/CompactFER#1]
Hi! yh1226!  Your code is helpful for me !  Thank you for sharing!  But i have a question about your code !  In you code, `model.fit(x_train,y_train, batch_size=32, epochs=50,validation_data=(x_test,y_test), verbose=1, shuffle=True,callbacks=callback_lists)` means,i guess,you train your model in training data, and test in testing data, and  78% is your best accuracy that 'keras' select in testing data!  But my thought is you can split your training data into training data and validation data, and select the best model that can get the highest accuracy in validation data, then test in testing data . The accuracy in testing data is the 'real' test accuracy!  I guess the accuracy may be lower than 78%!  However,  i am a beginner about 'keras' or 'deep learning'. I just have this question about your code. Maybe you are right. In the end, thanks again for your sharing !

**Comment by YanGuangyou** [comment]
> Hi! yh1226! Your code is helpful for me ! Thank you for sharing! But i have a question about your code ! In you code, `model.fit(x_train,y_train, batch_size=32, epochs=50,validation_data=(x_test,y_test), verbose=1, shuffle=True,callbacks=callback_lists)` means,i guess,you train your model in training data, and test in testing data, and 78% is your best accuracy that 'keras' select in testing data! But my thought is you can split your training data into training data and validation data, and select the best model that can get the highest accuracy in validation data, then test in testing data . The accuracy in testing data is the 'real' test accuracy! I guess the accuracy may be lower than 78%! However, i am a beginner about 'keras' or 'deep learning'. I just have this question about your code. Maybe you are right. In the end, thanks again for your sharing !

Hello,I think your thought is right!The 78% is higher than the paper.Do you have the improved code?If permissible,can you share to me?
